ramlfications/utils/types.py

```diff
@@ -119,17 +119,10 @@ def parse_example(name, node):
     data = dict(name=name, value=node)
     if name:
         data["display_name"] = name
-    if isinstance(node, dict):
-        # Might have a 'value' key; adds a layer.
-        if "value" in node:
-            data["value"] = node["value"]
-            if "description" in node:
-                data["description"] = BaseContent(node["description"])
-            if "displayName" in node:
-                data["display_name"] = node["displayName"]
-            if "strict" in node:
-                data["strict"] = node["strict"]
-    if "display_name" not in data:
-        data["display_name"] = name
+    if isinstance(node, dict) and "value" in node:
```

Comment:
Not in this case.  When there's a ``value`` key, the entire interpretation changes.  If there isn't a value key, we *already* have the value, and we don't want to clobber it.  Only when we have the value key can the other bits of metadata be present.